HIP/hip-873.md

```diff
@@ -0,0 +1,103 @@
+---
+hip: 873
+title: Adding Decimal Information to Token Balance Calls in the Hedera REST API
+author: Houdini NFT (@HoudiniNFT), Ty Smith (@ty-swirldslabs)
+requested-by: Atlantis
+type: Standards Track
+category: Mirror
+needs-council-approval: Yes
+status: Review
+created: 2024-01-30
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/874
+updated: 2024-01-31
+---
+
+### Abstract
+This HIP proposes an enhancement (with backwards compatibility) to the Hedera REST API to include decimal information in the token balance queries. Currently, users face challenges in accurately determining token balances due to the absence of decimal details in the `/tokens` and `/account/{id}/tokens` endpoints. This proposal aims to reduce the number of API calls needed and streamline the process of retrieving comprehensive token information.
+
+### Motivation
+Retrieving token balance information without decimal details requires users to make multiple API calls, leading to inefficiencies and increased computational load. Decimals should be supplied with the balance amount. Specifically, users must perform additional queries to obtain the decimal count for each token associated with an account. This is crucial for determining the correct token balance. This process is cumbersome, time-consuming, and can lead to increased costs and latency in applications relying on the Hedera API.
```

Comment:
We should mention hip-367 and how it is deprecating the HAPI queries and we're ensuring that the mirror node APIs have parity with those. Also add hip-367 to a references section and corresponding mirror node ticket.

Reply:
Added something about these things. Let me know if it suffices